expo/gbe/cms_app.py

```diff
@@ -2,9 +2,10 @@
 from cms.apphook_pool import apphook_pool
 from django.utils.translation import ugettext_lazy as _
 
-#this is largely a proof of concept right now, I haven't found this this was useful.
+
+# this is largely a proof of concept right now,
```

Comment:
The only reason this was touched was pep8 fixing.
The code is not in use on the site, but it IS a useful stub if we address the desire to put forms within CMS pages - this is the foundation of that, but it was never finished.